src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstall.groovy

```diff
@@ -120,6 +119,16 @@ class PaketUnityInstall extends ConventionTask {
         project.files(files)
     }
 
+    PaketUnityInstall() {
+        description = 'Copy paket dependencies into unity projects'
+        group = PaketUnityPlugin.GROUP
+        includeAssemblyDefinitions = false
+    }
```

Comment:
We don't put this information in the classes. Because the group and description, depending on the actual uses can change or can be set to another value. That's why we keep this in the pluging class when creating the tasks.

Reply:
Oh this was here already; I just moved the constructor after the properties. I could refactor it into the plugin...

Reply:
Then not to worry.